src/service/webhook

```diff
@@ -1,250 +1,279 @@
+            help=argparse.SUPPRESS,
+        )
+        parser.add_argument(
+            '--token',
+            default='',
+            help=argparse.SUPPRESS,
+        )
+        parser.add_argument(
+            '--debian',
+            choices=['Y', 'N'],
+            default='N',
+            help=argparse.SUPPRESS,
+        )
+        parser.add_argument(
+            '--dumb',
+            choices=['Y', 'N'],
+            default='N',
+            help=argparse.SUPPRESS,
+        )
+        parser.add_argument(
+            '--build',
+            choices=['true', 'false'],
+            default='true',
+            help=argparse.SUPPRESS,
+        )
+        parser.add_argument(
+            '--notify',
+            choices=['true', 'false'],
+            default='true',
+            help=argparse.SUPPRESS,
+        )
+        parser.add_argument(
+            '--comment',
+            default='',
+            help=argparse.SUPPRESS,
+        )
+        self.args = parser.parse_args(args=args)
+
+    @property
+    def old_whm(self):
+        if not hasattr(self, '_old_whm'):
+            params = {
+                'obs': self.OBS,
+                'project': self.project,
+                'package': self.package,
+                #'repourl': self.args.repourl,
+                #'branch': self.args.branch,
+            }
+            self._debug('Fetching old whm')
+            self._debug(params)
+            r = requests.get(
+                '%s/webhookmappings/' % self.WEBHOOK_URL,
+                auth=self.auth,
+                params=params,
+            )
+            r.raise_for_status()
+            response = r.json()
+            self._debug(response)
+            if response['count'] > 1:
+                raise Exception(
+                    'More than one webhook mapping found for %s' % (params,)
+                )
+            elif response['count'] == 0:
+                self._debug('No old whm found')
+                self._old_whm = None
+            else:
+                self._old_whm = response['results'][0]
+                self._debug('Got old whm')
+                self._debug(self._old_whm)
+        return self._old_whm
+
+    def update_old_whm(self):
+        self._debug("Updating old whm")
+        r = requests.patch(
+            '%s/webhookmappings/%s/' % (self.WEBHOOK_URL, self.old_whm['id']),
+            auth=self.auth,
+            json=self.new_whm,
+        )
+        r.raise_for_status()
+        result = r.json()
+        self._debug(result)
+        return result
+
+    def create_new_whm(self):
+        self._debug("Creating new whm")
+        r = requests.post(
+            '%s/webhookmappings/' % self.WEBHOOK_URL,
+            auth=self.auth,
+            json=self.new_whm,
+        )
+        r.raise_for_status()
+        result = r.json()
+        self._debug(result)
+        return result
+
+    def trigger_whm(self, whm):
+        self._debug("Triggering whm %s" % whm['id'])
```

Comment:
how about 
self._debug("Triggering whm %s for %s %s on %s" % (whm['id'],  whm['project'],  whm['package'], whm['obs'])